src/functionality.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-console */
+// Function to fetch leaderboard data from the API and populate the table
+async function fetchLeaderboard() {
+  try {
+    // eslint-disable-next-line no-use-before-define
+    const data = await fetchLeaderboardFromAPI();
+    const leaderboardBody = document.getElementById('leaderboard-body');
+    leaderboardBody.innerHTML = ''; // Clear existing data
+
+    data.forEach((entry) => {
+      const row = document.createElement('tr');
+      row.innerHTML = `
+          <td>${entry.name}</td>
+          <td>${entry.score}</td>
+        `;
+      leaderboardBody.appendChild(row);
+    });
+  } catch (error) {
+    console.error('Error fetching leaderboard data:', error);
+  }
+}
+
+// Function to fetch leaderboard data from the API (Mock data for this example)
+function fetchLeaderboardFromAPI() {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      const mockData = [
+        { name: 'Player 1', score: 1000 },
+        { name: 'Player 2', score: 900 },
```

Comment:
Good job working on the project requirements. Its against microverse policy to disable eslint or any other linters because linters help us write clean code. it would be nice if you enable eslint.